tests/test_dst_components.py

```diff
@@ -0,0 +1,351 @@
+# SPDX-FileCopyrightText: ASSUME Developers
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+import pandas as pd
+import pyomo.environ as pyo
+import pytest
+
+from assume.units.dst_components import (
+    DriPlant,
+    DRIStorage,
+    ElectricArcFurnace,
+    Electrolyser,
+    GenericStorage,
+)
+
+
+# Test Electrolyser class
+@pytest.fixture
+def electrolyser_unit() -> Electrolyser:
+    return Electrolyser(
+        model=pyo.ConcreteModel(),
+        id="Test_Electrolyser",
+        rated_power=100,
+        min_power=10,
+        ramp_up=20,
+        ramp_down=20,
+        min_operating_time=1,
+        min_down_time=1,
+        efficiency=0.8,
+        fuel_type="Electricity",
+        index=pd.date_range("2022-01-01", periods=4, freq="h"),
+    )
+
+
+def test_init_function(electrolyser_unit):
```

Comment:
This file has a ton of functions with the same name, this won't work and will always raise an error. Please fix